### DISCUSSION TITLE
How to set the default view of a bigWig track to line

### DISCUSSION THREAD

**Discussion by zengXchao** [GMOD/jbrowse-components#3712] [Q&A] [answered]
Hello, I would like to set the default parameters for a bigWig track as follows:

1. Renderer type: line
2. Min score: 0
3. Max score: 1

My configuration file is as follows, but it still shows xyplot, and the max/min score values are not changed, how can I fix it?

```
{
  "type": "QuantitativeTrack",
  "trackId": "bigWig_test",
  "name": "Test",
  "assemblyNames": [
    "hg38"
  ],
  "category": [
    "Test"
  ],
  "adapter": {
    "type": "BigWigAdapter",
    "bigWigLocation": {
      "locationType": "UriLocation",
      "uri": "data/test.bw"
    }
  },
  "displays": [
    {
      "displayId": "bigWig_test-LinearWiggleDisplay",
      "type": "LinearWiggleDisplay",
      "renderer": {
        "type": "LinePlotRenderer"
      },
      "rendererTypeNameState": "line",
      "height": 50,
      "resolution": 1,
      "constraints": {
        "max": 1,
        "min": 0
      }
    }
  ]
}
```

**Comment by cmdcolin** [comment, marked as answer]
Here is a track that uses the line plot, and adds the 'height' and minScore/maxScore parameters

```json
{
  "type": "QuantitativeTrack",
  "trackId": "volvox_microarray_line-1684526501696-sessionTrack",
  "name": "wiggle_track lineplot (copy)",
  "assemblyNames": [
    "volvox"
  ],
  "adapter": {
    "type": "BigWigAdapter",
    "bigWigLocation": {
      "locationType": "UriLocation",
      "uri": "volvox_microarray.bw"
    }
  },
  "displays": [
    {
      "type": "LinearWiggleDisplay",
      "displayId": "volvox_microarray_line_line-1684526501696",
      "height": 500,
      "minScore": 0,
      "maxScore": 1,
      "defaultRendering": "line"
    }
  ]
}
```

The settings are admittedly a bit tricky to configure, but one way to help is to edit the config in the user interface (specifically with the "Settings" sidebar, not with the 'Track menu' which does not edit the config itself)

And then, go to "About track" and click "Copy config"

![image](https://github.com/GMOD/jbrowse-components/assets/6511937/4f50ad24-d8ba-4e10-9bb7-32d3bf7a2bbe)

**Reply by zengXchao** [reply]
It is working for me, thank you very much!
